<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="ExcelTab">
      <attribute name="title">
       <string>Excel</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="Excel_TableName">
           <property name="placeholderText">
            <string>TableName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ExcelBrowse_pushButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="ExcelHeaders_checkBox">
         <property name="text">
          <string>Headers</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QRadioButton" name="ExcelReplace_radioButton">
           <property name="text">
            <string>Replace</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExcelReplaceSheet_radioButton">
           <property name="text">
            <string>Replace sheet</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ExcelAppendSheet_radioButton">
           <property name="text">
            <string>Append sheet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="ExcelWorksheetName_lineEdit">
         <property name="placeholderText">
          <string>Worksheetname</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ExcelAppendSync_checkBox">
         <property name="toolTip">
          <string>This option will find unique values in column in current loaded data,
delete all rows from worksheet which contain this values 
and append current data.</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Synchronize/update data using column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ExcelSyncColumn_comboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="CSVTab">
      <attribute name="title">
       <string>CSV</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLineEdit" name="CSV_TableName">
           <property name="placeholderText">
            <string>TableName</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="CSVBrowse_pushButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="CSVHeaders_checkBox">
         <property name="text">
          <string>Headers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Delimeter</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QRadioButton" name="CSVDelimeterTab_radioButton">
           <property name="text">
            <string>Tab</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="CSVDelimeterSpace_radioButton">
           <property name="text">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="CSVDelimeterSemicolumn_radioButton">
           <property name="text">
            <string>;</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="CSVDelimeterComma_radioButton">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QRadioButton" name="CSVReplace_radioButton">
              <property name="text">
               <string>Replace</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <property name="autoExclusive">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="CSVAppend_radioButton">
              <property name="text">
               <string>Append</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DatabaseTab">
      <attribute name="title">
       <string>Database</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Connection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="DBConnection_comboBox"/>
       </item>
       <item>
        <widget class="QLineEdit" name="DB_TableName">
         <property name="placeholderText">
          <string>TableName</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="DBReplace_radioButton">
           <property name="text">
            <string>Replace</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="DBAppend_radioButton">
           <property name="text">
            <string>Append</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="Export_pushButton">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
